crypto/asn1/bio_asn1.c

```diff
@@ -16,6 +16,7 @@
 #include <string.h>
 #include <internal/bio.h>
 #include <openssl/asn1.h>
+#include "e_os.h"
```

Comment:
On side note, one can wonder if `# define ossl_assert(x) (int)(x)` or  `ossl_assert_int((int)(x),...` is right. Thing about explicit cast is that it will mask warnings than might be useful or even produce false positives. As an extreme example consider `if (ossl_assert(ll))`, where ll is non-zero 64-bit value with least significant 32 bits zero. Correct syntax for this would be `if (ossl_asser(ll != 0)`, and thing here is that explicit cast leaves *no* chance for compiler to issue warning.